[CI] formatting with black 26.1.0

`black` `26.1.0` introduced several changes in formatting which trigger some errors:

```diff
--- /home/runner/work/tox-console-scripts/tox-console-scripts/src/tox_console_scripts/plugin.py	2026-01-19 07:25:20.634522+00:00
+++ /home/runner/work/tox-console-scripts/tox-console-scripts/src/tox_console_scripts/plugin.py	2026-01-19 07:25:32.800292+00:00
@@ -5,11 +5,10 @@
 
 from pyproject_installer.lib.normalization import pep503_normalized_name
 from pyproject_installer.lib.scripts import generate_entrypoints_scripts
 
 from tox.plugin import impl
-
 
 logger = logging.getLogger(__name__)
 
 
 def distr_name(distr):
--- /home/runner/work/tox-console-scripts/tox-console-scripts/tests/test_plugin.py	2026-01-19 07:25:20.634522+00:00
+++ /home/runner/work/tox-console-scripts/tox-console-scripts/tests/test_plugin.py	2026-01-19 07:25:32.843416+00:00
@@ -64,13 +64,11 @@
     - check the generated script
     """
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         commands = {expected_console_script} --help
         """
@@ -105,13 +103,11 @@
     - check if console script was not generated
     """
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         commands=python -c "print('Hello, hello')"
         """
@@ -140,13 +136,11 @@
     - check if console script was not generated
     """
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
         # don't build package (no available build backend)
         package = skip

Oh no! 💥 💔 💥
2 files would be reformatted, 4 files would be left unchanged.
Error: Process completed with exit code 1.
```

See for details:
https://black.readthedocs.io/en/stable/change_log.html

--- a/.github/workflows/pipelines.yml
+++ b/.github/workflows/pipelines.yml
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v5
@@ -49,7 +49,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', '3.14']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v5

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "tox-console-scripts"
 description = "Create console scripts of system site packages within tox env"
-requires-python = ">= 3.9"
+requires-python = ">= 3.10"
 dependencies = [
   "tox > 4",
   "pyproject-installer >= 0.4.0",
@@ -31,7 +31,6 @@ classifiers = [
   "Topic :: Utilities",
   "Programming Language :: Python :: 3",
   "Programming Language :: Python :: 3 :: Only",
-  "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",

--- a/src/tox_console_scripts/plugin.py
+++ b/src/tox_console_scripts/plugin.py
@@ -8,7 +8,6 @@ from pyproject_installer.lib.scripts import generate_entrypoints_scripts
 
 from tox.plugin import impl
 
-
 logger = logging.getLogger(__name__)
 
 

--- a/tests/test_plugin.py
+++ b/tests/test_plugin.py
@@ -66,9 +66,7 @@ def test_plugin_usage(tox_project):
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
@@ -107,9 +105,7 @@ def test_no_plugin_usage(tox_project):
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]
@@ -142,9 +138,7 @@ def test_plugin_usage_no_systemsite(tox_project):
     project = tox_project()
     expected_console_script = "pytest"
     env_name = "py"
-    project.contents[
-        "tox.ini"
-    ] = f"""\
+    project.contents["tox.ini"] = f"""\
         [tox]
         env_list = {env_name}
         [testenv]

